composer.json

```
@@ -25,18 +25,18 @@
         "ext-json": "*",
         "barryvdh/reflection-docblock": "^2.3",
         "composer/class-map-generator": "^1.0",
-        "illuminate/console": "^11.15",
-        "illuminate/database": "^11.15",
-        "illuminate/filesystem": "^11.15",
-        "illuminate/support": "^11.15"
+        "illuminate/console": "^11.15|^12",
+        "illuminate/database": "^11.15|^12",
+        "illuminate/filesystem": "^11.15|^12",
+        "illuminate/support": "^11.15|^12"
     },
     "require-dev": {
         "ext-pdo_sqlite": "*",
         "friendsofphp/php-cs-fixer": "^3",
-        "illuminate/config": "^11.15",
-        "illuminate/view": "^11.15",
+        "illuminate/config": "^11.15|^12",
+        "illuminate/view": "^11.15|^12",
         "mockery/mockery": "^1.4",
-        "orchestra/testbench": "^9.2",
+        "orchestra/testbench": "^9.2|^10",
         "phpunit/phpunit": "^10.5",
```

CI is reporting the [following error](https://github.com/barryvdh/laravel-ide-helper/actions/runs/13214322327/job/36892356377?pr=1672):

```
orchestra/testbench 10.x-dev requires phpunit/phpunit ^11.5.3
```

So the following should fix the dependency issue:

```suggestion
        "phpunit/phpunit": "^10.5|^11.5.3",
```